lib/gitlab/client/runners.rb

```diff
@@ -0,0 +1,124 @@
+class Gitlab::Client
+  # Defines methods related to runners.
+  # @see https://github.com/gitlabhq/gitlabhq/blob/master/doc/api/runners.md
+  module Runners
+
+    # Get a list of specific runners available to the user.
+    # Full runner params documentation: https://github.com/gitlabhq/gitlabhq/blob/master/doc/api/runners.md
+    #
+    # @example
+    #   Gitlab.runners
+    #   Gitlab.runners(:active)
+    #   Gitlab.runners(:paused)
+    #
+    # @param  [Hash] options A customizable set of options.
+    # @option options [String] :scope The scope of specific runners to show, one of: active, paused, online; showing all runners if none provided
+    # @return [Array<Gitlab::ObjectifiedHash>]
+    def runners(options = {})
+      if options[:scope]
+        get("/runners", query: options)
+      else
+        get("/runners")
+      end
```

Comment:
Do we need the check here?  Couldn't we just pass the empty hash down regardless with no ill effect?